cdm-test/src/test/java/ucar/nc2/TestCharFillValue.java

```diff
@@ -0,0 +1,83 @@
+/*********************************************************************
```

Comment:
@DennisHeimbigner - sorry for the minor comment here, but could we use the netCDF-Java / TDS copyright block in this?

~~~java
 /*
  * Copyright (c) 1998-2018 University Corporation for Atmospheric Research/Unidata
  * See LICENSE for license information.
  */
~~~

Reply:
Sure, go ahead and change it.